cheatsheets/Microservices_based_Security_Arch_Doc_Cheat_Sheet.md

```diff
@@ -0,0 +1,245 @@
+* "Identify and describe infrastructure services";
+* "Identify and describe data storages";
+* "Identify and describe message queues".
+
+#### Mapping to OWASP projects
+
+* [OWASP ASVS, V1 “Architecture, Design and Threat Modeling Requirements”, #1.1.5](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x10-V1-Architecture.md#v1-architecture-design-and-threat-modeling-requirements)
+
+### Implementation of centralized security controls verification
+
+#### Implementation tips
+
+To verify implementation of centralized, simple (economy of design), vetted, secure, and reusable security controls to avoid duplicate, missing, ineffective, or insecure controls analyze data collected under the section "Identify and describe infrastructure services".
+
+#### Mapping to OWASP projects
+
+* [OWASP ASVS, V1 “Architecture, Design and Threat Modeling Requirements”, #1.1.6](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x10-V1-Architecture.md#v1-architecture-design-and-threat-modeling-requirements)
+
+### Enforcement of the principle of least privilege
+
+#### Implementation tips
+
+To define minimally needed microservice permissions analyze data collected under the following sections:
+* "Identify and describe application-functionality services (parameter “API definition”)";
+* "Identify “service-to-storage” relations";
+* "Identify “service-to-service” synchronous communications"; 
+* "Identify “service-to-service” asynchronous communications".
+
+#### Mapping to OWASP projects
+
+* [OWASP ASVS, V1 “Architecture, Design and Threat Modeling Requirements”, #1.4.3](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x10-V1-Architecture.md#v1-architecture-design-and-threat-modeling-requirements)
+
+### Sensitive data identification and classification
+
+#### Implementation tips
+
+To verify that all sensitive data is identified and classified into protection levels analyze data collected under the following sections:
+* "Identify and describe data assets";
+* "Identify “asset-to-storage” relations".
+
+#### Mapping to OWASP projects
+
+* [OWASP ASVS, V1 “Architecture, Design and Threat Modeling Requirements”, #1.8.1](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x10-V1-Architecture.md#v1-architecture-design-and-threat-modeling-requirements)
+
+### Application components business/security functions verification
+
+#### Implementation tips
+
+To verify the definition and documentation of all application components in terms of the business or security functions they provide analyze data collected under the following sections (parameter “Short description”): 
+* "Identify and describe application-functionality services";
+* "Identify and describe infrastructure services".
+
+#### Mapping to OWASP projects
+
+* [OWASP ASVS, V1 “Architecture, Design and Threat Modeling Requirements”, #1.11.1](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x10-V1-Architecture.md#v1-architecture-design-and-threat-modeling-requirements)
+
+# Authors and Primary Editors

```

Comment:
Kindly remove this section as we are relying on the Github functionality for the contributors

Reply:
Done